app/models/agents/website_agent.rb

```diff
@@ -166,6 +169,27 @@ def validate_options
       validate_web_request_options!
     end
 
+    def validate_consider_http_success_option!
+      consider_success = options["http_success_codes"]
+      if consider_success != nil
```

Comment:
I'm not sure I understand the part with `consider_success.present?`. The way I imagined configuring the error codes, is an array of codes, like:
`http_success_codes: [404, 422]` . Does this mean, that you expect `http_success_codes: []` to be valid, and then rely on `response.success?`